checks/evaluation/security_policy.go

```diff
@@ -39,11 +40,14 @@ func SecurityPolicy(name string, dl checker.DetailLogger, r *checker.SecurityPol
 		}
 		if msg.Type == checker.FileTypeURL {
 			msg.Text = "security policy detected in org repo"
+			dl.Info(&msg)
 		} else {
-			msg.Text = "security policy detected"
+			// Security policy detected in current repo, return immediately since it has higher priority.
+			msg.Text = "security policy file detected in current repo"
+			dl.Info(&msg)
+			return checker.CreateMaxScoreResult(name, "security policy file detected in current repo")
```

Comment:
you can keep the code the same in each if-statements. No need to return here. Since we allow multiple files for the repo, let's do the same for org repos.

Reply:
Updated, thanks